server/api.js

```diff
@@ -115,15 +115,13 @@ router.post("/chat", (req, res) => {
   if (req.body.recipient._id == "ALL_CHAT") {
     socket.getIo().emit("chat", message);
   } else {
-    const recipientSocketID = socket.getSocketFromUserID(req.body.recipient._id);
     socket
       .getIo()
-      .to(recipientSocketID)
+      .to(req.body.recipient._id)
```

Comment:
what are your thoughts on making a variable that is like `const recipientRoomId = req.body.recipient._id;` and using that so people looking over it understand just from naming that rooms are being used?